Make skip-errors more visible (#18385)
sourcegraph/sourcegraph: doc/campaigns/how-tos/creating_a_campaign.md

    <pre><code>src campaign preview -f <em>YOUR_CAMPAIGN_SPEC.campaign.yaml</em></code></pre>

    > **Don't worry!** Before any branches are pushed or changesets (e.g., GitHub pull requests) are created, you will see a preview of all changes and can confirm each one before proceeding.

>     > NOTE: Campaigns's default behavior is to stop if creating changes on a repository errors. You can choose to ignore errors instead by adding the [`skip-errors`](../../cli/references/campaigns/preview.md) flag : `src campaign preview -f spec.campaign.yml -skip-errors`

Reviewer: ```suggestion
    > NOTE: Campaigns's default behavior is to stop if computing changes in a repository errors. You can choose to ignore errors instead by adding the [`skip-errors`](../../cli/references/campaigns/preview.md) flag : `src campaign preview -f spec.campaign.yml -skip-errors`
```
